src/test/java/com/iexec/common/sdk/order/payload/RequestOrderTest.java

```diff
@@ -0,0 +1,67 @@
+package com.iexec.common.sdk.order.payload;
+
+import com.iexec.common.chain.DealParams;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+import java.util.Map;
+
+class RequestOrderTest {
+
+    @Test
+    void shouldSerializeEmptyMapWhenNoParameters() {
+        DealParams params = DealParams.builder().build();
+        Assertions.assertEquals("{}", RequestOrder.toStringParams(params));
+    }
+
+    @Test
+    void shouldSerializeEmptyMapWhenBooleanParametersAreFalse() {
+        DealParams params = DealParams.builder()
+                .iexecResultEncryption(false)
+                .iexecDeveloperLoggerEnabled(false)
+                .build();
+        Assertions.assertEquals("{}", RequestOrder.toStringParams(params));
+    }
+
+    @Test
+    void shouldSerializeParametersWithEncryptedResult() {
+        for (String storageProxy : List.of(DealParams.DROPBOX_RESULT_STORAGE_PROVIDER, DealParams.IPFS_RESULT_STORAGE_PROVIDER)) {
```

Comment:
What about a parametrized test there, instead of a `for` loop?

Reply:
Fixed in https://github.com/iExecBlockchainComputing/iexec-common/pull/298/commits/7107a8db754fd4807bf1ec149afd78fa41506bd1